src/rules/trailingCommaRule.ts

```diff
@@ -230,4 +230,16 @@ class TrailingCommaWalker extends Lint.AbstractWalker<Options> {
             this.addFailureAt(list.end - 1, 1, Rule.FAILURE_STRING_NEVER, Lint.Replacement.deleteText(list.end - 1, 1));
         }
     }
+
+    private isWrappedSingleExpression(list: ts.NodeArray<ts.Node>) {
+        if (list.length !== 1) {
+            return false;
+        }
+
+        const parentNodeChildren = list[0].parent!.getChildren();
+        const openTokenIndex = parentNodeChildren.findIndex((node) => node.pos === list.pos) - 1;
+        const openToken = parentNodeChildren[openTokenIndex];
+
+        return isSameLine(this.sourceFile, openToken.end, list[0].getStart());
```

Comment:
when checking for "wrapped" expressions you also need to check if the expression ends on (or after) the next line.

Consider the following code which would pass with the current implementation:
```ts
foo(bar // trailing comma should be required here
);
```